sdk/attestation/attestation/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "module": "es6",
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es5",
```

Comment:
do you need to support ie9? if not you can pick at least es6 here

Reply:
Done similar to search-documents